### PR TITLE
Add Fragbite class

### PR DESCRIPTION
**Pull request by ruddan**
Adds a reader for Fragbite.se

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,10 +4,15 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
+pylint = "*"
+black = "*"
```
Comment (bombsimon) [inline Pipfile]: Nice! ✅

File: scraper.py [hunk 2]
```diff
@@ -285,3 +285,37 @@ def source(self):
...
+
+            result.append(
+                header.Header(
+                    article.h1.get_text(),
+                    subheading.get_text(),
+                    "{}{}".format(self.url(), article.a.get("href")),
+                    False,
+                )
+            )
+        return result
```
Comment (bombsimon) [inline scraper.py]: Mind adding a newline for consistency?
```suggestion
            )
            
        return result
```

**Review by bombsimon — APPROVED** [review]
LGTM! If you just solve the merge conflicts and the small comment I made I'll gladly merge!

File: scraper.py (hunk 2 above)
Comment (ruddan) [reply scraper.py]: Done!

**Pull request merged**